src/test_IndexGame.py

```diff
@@ -103,3 +103,18 @@ def test_buildBuildings(map, selectedBuilding, buildingPool, turn, input, expect
     result = IndexGame.buildBuildings(map, selectedBuilding, buildingPool, turn, input)
     assert result == expected
     
+
+def test_randomBuilding():
+    availabletypes = ['BCH', 'HSE', 'SHP', 'FAC', 'MON']
+    result = IndexGame.randomBuilding({
+                "BCH":8, 
+                "HSE":8,
+                "SHP":8,
+                "FAC":8,
+                "HWY":0,
+                "MON":8,
+                "PRK":0
+            })
+    expected = "BCH", "FAC"
```

Comment:
Hi, I think there is a logical error, cause there will be multiple outputs from this scenario. Might need to check if result in list instead.